server/routes/booster-generation/sets/jou.js

```diff
@@ -0,0 +1,256 @@
+    "colors": [
+      "B",
+      "U"
+    ]
+  },
+  {
+    "id": "3184b138-1109-4195-9d96-4f190164e98b",
+    "name": "Xenagos, God of Revels",
+    "image_uris": {
+      "large": "https://img.scryfall.com/cards/large/front/3/1/3184b138-1109-4195-9d96-4f190164e98b.jpg?1562502824"
+    },
+    "cmc": 5,
+    "oracle_text": "Indestructible\nAs long as your devotion to red and green is less than seven, Xenagos isn't a creature.\nAt the beginning of combat on your turn, another target creature you control gains haste and gets +X/+X until end of turn, where X is that creature's power.",
+    "colors": [
+      "G",
+      "R"
+    ]
+  }
+];
+
+const ratio_of_god_packs = 1080;
+
+function getRandomIndex(length = 0) 
+{
+    return Math.floor(Math.random() * length);
+}
+
+function generatePacks(cards, count, lands, mapCard)
+{
+    const boosters = [];
+    const commons = cards.filter(card => card.rarity === 'common');
+    const uncommons = cards.filter(card => card.rarity === 'uncommon');
+    let rares = cards.filter(card => card.rarity === 'rare');
+    rares = rares.concat(rares);
+    rares.concat(cards.filter(card => card.rarity === 'mythic'));
+
+    while( boosters.length < count )
+    {
+        if( getRandomIndex(ratio_of_god_packs) === 540 )
+        {//create a god pack
+            let booster = [...god_pack];
+            //put the land in 
+            booster.push(lands[getRandomIndex(lands.length)]);
+            //Unpackages the card
+            boosters.push(booster.map(mapCard));
+        }
+        else
+        {
+            let booster = [];
+            //create a boolean to indicate if we have a foil
+            const isFoil = getRandomIndex(6) === 2;
+            //populate the commons
+            for(let i = 0; i < (isFoil ? 10 : 11); i++) 
+            {
+                booster.push(commons[getRandomIndex(commons.length)]);
+            }
+            //populate the uncommons
+            for(let i = 0; i < 3; i++) 
+            {
+                booster.push(uncommons[getRandomIndex(uncommons.length)]);
+            }
+            //populate the rares and mythics
+            booster.push(rares[getRandomIndex(rares.length)]);
+            //fill in the foil cards
+            if (isFoil) 
+            {
+                booster.push(
+                {
+                    ...cards[getRandomIndex(cards.length)],
+                    isFoil: true
+                });
+            }
+            //put the land in 
+            booster.push(lands[getRandomIndex(lands.length)]);
+            //Unpackages the card
+            boosters.push(booster.map(mapCard));
+        }
+    }
+    return boosters;
+}
+
+exports.generatePacks = generatePacks; 
```

Comment:
missing an empty new line at the end of the file. ES6 style standard ;p

Reply:
Another style comment is that this seems to use 4 spaces for indentation. This project uses 2.

Reply:
These aren't essential and I can merge this and then fix the style. Just good to know for future PRs